packages/nextjs/tailwind.config.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
           "base-100": "#D8DCFF",
           "base-200": "#FFD4B5",
           "base-300": "#F4F4F4",
+          "base-400": "#EBECFD",
```

Comment:
Ideally this shouldn't work 😅  I think we only use this color names in theme : 
https://v2.daisyui.com/docs/colors/#-2 which does not have `base-400` I think this is the reason why we added again `base-400` in `backgroundColor` ?

Reply:
Ups! I think I understand now. For daisyUI we just update existing daisyUI classes, and in `theme` section we can create new tailwind classes to use them in our code. I didn't have it very clear as you can see 😅 

That comes from my daisyUI v2 `table` tests and forgot to delete it.

Fixed now 🙌